Pod/Classes/Optional Data Sources/VOKCollectionDataSource.h

```diff
@@ -9,51 +10,124 @@
-     sectionNameKeyPath:(nullable NSString *)sectionNameKeyPath
-        sortDescriptors:(nullable VOKArrayOfSortDescriptors *)sortDescriptors
-     managedObjectClass:(Class)managedObjectClass
-               delegate:(nullable id <VOKFetchedResultsDataSourceDelegate>)delegate;
-
-- (id)initWithPredicate:(nullable NSPredicate *)predicate
-              cacheName:(nullable NSString *)cacheName
-         collectionView:(nullable UICollectionView *)collectionView
-     sectionNameKeyPath:(nullable NSString *)sectionNameKeyPath
-        sortDescriptors:(nullable VOKArrayOfSortDescriptors *)sortDescriptors
-     managedObjectClass:(Class)managedObjectClass
-              batchSize:(NSInteger)batchSize
-               delegate:(nullable id <VOKFetchedResultsDataSourceDelegate>)delegate;
-
-- (id)initWithPredicate:(nullable NSPredicate *)predicate
-              cacheName:(nullable NSString *)cacheName
-         collectionView:(nullable UICollectionView *)collectionView
-     sectionNameKeyPath:(nullable NSString *)sectionNameKeyPath
-        sortDescriptors:(nullable VOKArrayOfSortDescriptors *)sortDescriptors
-     managedObjectClass:(Class)managedObjectClass
-              batchSize:(NSInteger)batchSize
-             fetchLimit:(NSInteger)fetchLimit
-               delegate:(nullable id <VOKFetchedResultsDataSourceDelegate>)delegate;
+/**
+ Setup this data source.
+
+ @param predicate          Predicate used to filter the objects displayed
+ @param cacheName          Name for the fetched results controller cache
+ @param collectionView     Collection view in which results are shown. The data source and delegate are set to the newly-created data source.
+ @param sectionNameKeyPath Keypath to use to group results in sections
+ @param sortDescriptors    Sort descriptors to sort the objects
+ @param managedObjectClass NSManagedObject subclass to fetch
+
+ @return New data source
+ */
+- (instancetype)initWithPredicate:(nullable NSPredicate *)predicate
+                        cacheName:(nullable NSString *)cacheName
+                   collectionView:(nullable UICollectionView *)collectionView
+               sectionNameKeyPath:(nullable NSString *)sectionNameKeyPath
+                  sortDescriptors:(nullable VOKArrayOfSortDescriptors *)sortDescriptors
+               managedObjectClass:(Class)managedObjectClass;
+
+/**
+ Setup this data source.
+
+ @param predicate          Predicate used to filter the objects displayed
+ @param cacheName          Name for the fetched results controller cache
+ @param collectionView     Collection view in which results are shown. The data source and delegate are set to the newly-created data source.
+ @param sectionNameKeyPath Keypath to use to group results in sections
+ @param sortDescriptors    Sort descriptors to sort the objects
+ @param managedObjectClass NSManagedObject subclass to fetch
+ @param batchSize          Batch size to use for the fetch request fetchBatchSize. Defaults to 20.
```

Comment:
I think it makes more sense to put the default value comment on methods that _don't_ include the `batchSize` parameter.